autofill/autoprompt/1-standard-login-form.html

```diff
@@ -44,8 +44,9 @@ <h3>Sign in form</h3>
       e.preventDefault();
       e.stopImmediatePropagation();
       clickForm.outerHTML = '<h1>Submitted!</h1>';
-    }
+    };
     button?.addEventListener('click', handler, true);
+    clickForm?.addEventListener('submit', handler, true);
```

Comment:
I decided to add listeners to `submit` as well because on Android we're using `requestSubmit` and the handlers were not running on button click.